Dangerfile

```diff
@@ -33,5 +33,19 @@ if has_app_changes && missing_doc_changes && doc_changes_recommended
   warn("Consider adding supporting documentation to this change. Documentation can be found in the `docs` directory.")
 end
 
+# Warn when either the podspec or Cartfile + Cartfile.resolved has been updated,
+# but not both.
+podspec_updated = !git.modified_files.grep(/Moya.podspec).empty?
+cartfile_updated = !git.modified_files.grep(/Cartfile).empty?
+cartfile_resolved_updated = !git.modified_files.grep(/Cartfile.resolved).empty?
+
+if podspec_updated && !cartfile_updated && !cartfile_resolved_updated
```

Comment:
Shouldn't this be 

```ruby
if podspec_updated && (!cartfile_updated || !cartfile_resolved_updated)
```

 just like [line 46](https://github.com/Moya/Moya/blob/11680fccd9681942a5062beec9820d31e71741cb/Dangerfile#L46)?